common/app/views/fragments/bashers/ThisIsTheNHS.scala.html

```diff
@@ -0,0 +1,41 @@
+@()(implicit request: RequestHeader)
+
+<div class="gs-container">
+    <div class="this-is-the-nhs">
+        <h1 class="title title--first">this is</h1>
```

Comment:
@jbreckmckye very nice summary

Making atoms like title, stripe etc makes sense if they will be reused across multiple components (molecules). And then it should be split into separate files, eg. _stripe.scss so it will be easy to find.

However as this is probably very campaign specific and might be removed in the future, it will be much better to follow BEM 'single class' approach as @jbreckmckye suggested. It is very logical, easy to read, easy to find (one component) and there are no nesting/specificity issues involved.